demo/app.js

```diff
@@ -483,10 +487,21 @@ const Demo = React.createClass({
     })
   },
 
+  _handleTransitionClick (transitionType) {
```

Comment:
I think this is more of a toggle function. I would break it up into two functions because I feel like this function is doing two things. A function should only do one thing.
```javascript
_toggleSlideTransition () {
  this.setState({
    showSlideTransition: !this.state.showSlideTransition;
  })
},

_toggleFadeTransition () {
  this.setState({
    showFadeTransition: !this.state.showFadeTransition;
  })
}
```